streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql

```diff
@@ -281,22 +293,22 @@ create table "public"."t_flink_cluster" (
   "cluster_name" varchar(255) collate "pg_catalog"."default" not null,
   "options" text collate "pg_catalog"."default",
   "yarn_queue" varchar(100) collate "pg_catalog"."default",
-  "execution_mode" int2 not null,
+  "execution_mode" int2 not null default 1,
   "version_id" int8 not null,
   "k8s_namespace" varchar(255) collate "pg_catalog"."default",
   "service_account" varchar(50) collate "pg_catalog"."default",
   "description" varchar(255) collate "pg_catalog"."default",
   "user_id" int8,
   "flink_image" varchar(255) collate "pg_catalog"."default",
   "dynamic_options" text collate "pg_catalog"."default",
-  "k8s_rest_exposed_type" int2,
-  "k8s_hadoop_integration" boolean,
-  "flame_graph" boolean,
+  "k8s_rest_exposed_type" int2 default 2,
```

Comment:
Do we need to set a default value?

Reply:
> Do we need to set a default value?

Yes, the k8s_rest_exposed_type default value is  nodeport, which is also consistent with mysql-schema.sql

Reply:
ok